framework/include/variables/MooseVariableData.h

```diff
@@ -315,6 +315,20 @@ class MooseVariableData
     return _ad_u_dot;
   }
 
+  const ADTemplateVariableValue<OutputType> & adUDotDot() const
+  {
+    _need_ad = _need_ad_u_dotdot = true;
+
+    if (!_time_integrator)
+      // If we don't have a time integrator (this will be the case for variables that are a part
+      // of the AuxiliarySystem) then we have no way to calculate _ad_u_dotdot and we are just
+      // going to copy the values from _u_dotdot. Of course in order to be able to do that we need
+      // to calculate _u_dotdot
+      _need_u_dotdot = true;
+
+    return _ad_u_dotdot;
+  }
```

Comment:
I've been a guilty culprit in the past, but for things like this I'd prefer to keep the class definition clean of multi-line method definitions. You can put this outside the class definition as an `inline` function in the header

Reply:
Would you like me to move `adUDot` as well? That was here before I made these changes.

Reply:
You don't need to, but if you're pushing any more commits, feel free to

Reply:
Will do

Reply:
Do I need to be explicit about adding the `inline` to a method that isn't fully specialized? Take a look at 2fdf84b. The comments already present in `MooseVariableData.h` at the bottom of the file made that a little unclear.

Reply:
No you do not. I forgot that this was a templated class when I made my original comment. Any thing with a template parameter will be automatically `inline`